pokemonterminal/adapter/base.py

```diff
@@ -15,6 +15,13 @@ def set_image_file_path(self, image_file_path):
         """
         raise NotImplementedError()
 
+    def get_image_file_number(self):
+        """
+        Get the number of the background image of the terminal.
+        :rtype str
+        """
+        raise NotImplementedError()
```

Comment:
Yes, clever, was thinking of doing exactly like this. But instead of exception, print a error message and quit so you don't have to do all these stub methods in the other implementations.